src/olympia/versions/tasks.py

```diff
@@ -126,11 +156,30 @@ def extract_version_to_git(version_id, author_id=None, note=None):
     log.info('Extracting {version_id} into git backend'.format(
         version_id=version_id))
 
-    repo = AddonGitRepository.extract_and_commit_from_version(
-        version=version, author=author, note=note)
+    try:
+        repo = AddonGitRepository.extract_and_commit_from_version(
+            version=version, author=author, note=note)
 
-    log.info('Extracted {version} into {git_path}'.format(
-        version=version_id, git_path=repo.git_repository_path))
+        log.info('Extracted {version} into {git_path}'.format(
+            version=version_id, git_path=repo.git_repository_path))
+    except BrokenRefError as err:
```

Comment:
(The fact that we only handle `BrokenRefError` here is by design)

Reply:
Should add a comment about this.